ws-nextjs-app/pages/[service]/av-embeds/handleAvRoute.ts

```diff
@@ -125,9 +125,11 @@ export default async (context: GetServerSidePropsContext) => {
               promoSummary,
               type: AV_EMBEDS,
             },
+            embedded: true,
           }
         : null,
       pageType: AV_EMBEDS,
+      pathname: resolvedUrl,
```

Comment:
Really wicked spot! was wondering why it wasn't coming through in the RequestContext. ❤️